ReceiveMessage/specialMessageIds.ts

```diff
@@ -0,0 +1,15 @@
+const specialMessageIds = new Map();
```

Comment:
https://www.typescriptlang.org/docs/handbook/interfaces.html#function-types

Read this about function types and use generics to cast the map to be a `<string, custom function interface>` typing. The function should take in a userState and return a string, with the string being the responseString we're looking for.

Now, we can define functions for each special message within this file and attach them to the Map. Back in the `manageKeywordSent` function, all we have to do in the case a keyword is sent is to get the value (a function), call that function, and return whatever is returned by that function and everything should be all gucci.